planning/autoware_path_generator/src/node.hpp

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NODE_HPP_
+#define NODE_HPP_
```

Comment:
I think `NODE_HPP_` is too generic.

```suggestion
#ifndef AUTOWARE_PATH_GENERATOR_NODE_HPP_
#define AUTOWARE_PATH_GENERATOR_NODE_HPP_
```

Reply:
Since this is a private header file, it might not be something to worry about.